standard/classes.md

```diff
@@ -782,7 +782,9 @@ Nested types can be declared in multiple parts by using the `partial` modifier.
 >
 > *end example*
 
-The handling of attributes specified on the type or type parameters of different parts of a partial declaration is discussed in [§22.3](attributes.md#223-attribute-specification).
+The handling of attributes specified on the type or type parameters of different parts of a partial type declaration is discussed in [§22.3](attributes.md#223-attribute-specification).
+
+A partial type may contain partial member declarations ([§15.6.9](classes.md#1569-partial-methods)).
```

Comment:
See previous comment: Maybe move this paragraph up to line 727?

Reply:
@RexJaeschke to implement